get-started/index.md

```diff
@@ -67,42 +67,41 @@ packageserver-8cccc99dd-g7lkh       1/1     Running   0          3m3s
-`ClusterCryostat` Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
-2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the Cryostat
-Operator instance will only watch for `Cryostat` or `ClusterCryostat` instances created in that same Namespace.
+1. In **All Namespaces** installation mode, the **Cryostat Operator** instance will watch for **Cryostat** or
+**ClusterCryostat** Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
+2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the **Cryostat Operator** instance will only watch for **Cryostat** or **ClusterCryostat** instances created in that same Namespace.
 {% include howto_step.html
   summary="Install the Operator"
   image-name="cryostat-operatorhub-install-in-progress.png"
 %}
-Click "Install" and wait for the installation to complete. In this example we will proceed with **All Namespaces**.
+Click "*Install*" and wait for the installation to complete. In this example we will proceed with **All Namespaces**.
 
 Continue to [Setup](#setup).
 
 ## [Setup](#setup)
 
-**Note:** An alternative setup using the multi-namespace `ClusterCryostat` `CR` is described in
-[Alternate Setup](#alternate-setup). For simplicity we will continue with the single-namespace `Cryostat` `CR`.
+**Note:** An alternative setup using the multi-namespace **ClusterCryostat** `CR` is described in
+[Alternate Setup](#alternate-setup). For simplicity we will continue with the single-namespace **Cryostat** `CR`.
 
 ### [Deploying Cryostat](#deploying-cryostat)
 
 {% include howto_step.html
   summary="Create a Cryostat instance"
   image-name="cryostat-operatorhub-install-complete.png"
 %}
-Once the installation is complete, click **Create Cryostat** to create a `Cryostat` `CR` instance in an OpenShift
-Project (Kubernetes `Namespace`) of your choice. This provides configuration information for the Operator to know the
-specifics of how to deploy your Cryostat instance. For full details on how to configure the Cryostat deployment, see
+Once the installation is complete, click ***Create Cryostat*** to create a **Cryostat** `CR` instance in an **OpenShift
+Project** (Kubernetes `Namespace`) of your choice. This provides configuration information for the **Operator** to know the
+specifics of how to deploy your **Cryostat** instance. For full details on how to configure the **Cryostat** deployment, see
 [Configuring Cryostat](https://github.com/cryostatio/cryostat-operator/blob/{{site.data.versions.cryostat.release-branch}}/docs/config.md).
 
-If running Cryostat on Kubernetes and not OpenShift, you will also need to add Ingress configurations to your Cryostat
+If running **Cryostat** on **Kubernetes** and not **OpenShift**, you will also need to add **Ingress** configurations to your **Cryostat**
```

Comment:
Here it seems inconsistent between **Ingress** and `CR` - like I said in the other discussion thread, we should decide which one of these styles to apply, and then apply it evenly.

Reply:
Similarly a few lines higher, there is `Namespace`.